src/state/slices/preferencesSlice/preferencesSlice.ts

```diff
@@ -176,5 +179,14 @@ export const preferences = createSlice({
     setSnapInstalled(state, { payload }: { payload: boolean }) {
       state.snapInstalled = payload
     },
+    addWatchedAsset(state, { payload }: { payload: AssetId }) {
+      state.watchedAssets = [...state.watchedAssets, payload]
+    },
+    removeWatchedAsset(state, { payload }: { payload: AssetId }) {
+      state.watchedAssets = state.watchedAssets.filter(assetId => assetId !== payload)
+    },
+    setWatchedAssets(state, { payload }: { payload: AssetId[] }) {
+      state.watchedAssets = payload
+    },
```

Comment:
`addWatchedAsset` and `setWatchedAssets` could be consolidated into `setWatchedAssets`, with the payload being an array for consistency

Reply:
Maybe i'm lazy but having to then include two selectors anytime we want to add to this list, seems unnecessary. 

Much easier to just to add and remove with a singular assetId.